packages/react/src/__tests__/ReactElementValidator-test.js

```diff
@@ -317,6 +270,8 @@ describe('ReactElementValidator', () => {
         'or a class/function (for composite components) but got: null.' +
         (__DEV__ ? '\n\nCheck the render method of `ParentComp`.' : ''),
     );
+
+    // We can't use .toWarnDev() here because we can't chain it with .toThrowError()
     if (__DEV__) {
```

Comment:
Let’s bite the bullet and nest assertions?

Reply:
I'm not sure what you have in mind here. Mind giving a faux code example?

Reply:
```js
expect(() => {
  expect(() => {
  
  }).toWarnInDev()
}).toThrowError('message')
```

Although there's a hazard of not specifying the message and thus catching a failure from the inner expect.

Reply:
Maybe nesting them the other way around would work.

Reply:
Nesting `toWarnDev` inside of `toThrowError` won't work. Errors in `toWarnDev` are ignored. But nesting it the other way around (`toThrowError` inside) _does_ seem to work.